src/main/java/gov/usgs/traveltime/TTService.java

```diff
@@ -0,0 +1,276 @@
+package gov.usgs.traveltime;
+
+import gov.usgs.processingformats.TravelTimeData;
+import gov.usgs.processingformats.TravelTimeException;
+import gov.usgs.processingformats.TravelTimePlotDataBranch;
+import gov.usgs.processingformats.TravelTimePlotDataSample;
+import gov.usgs.processingformats.TravelTimePlotRequest;
+import gov.usgs.processingformats.TravelTimeReceiver;
+import gov.usgs.processingformats.TravelTimeRequest;
+import gov.usgs.processingformats.TravelTimeService;
+import gov.usgs.traveltime.tables.TauIntegralException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.NavigableMap;
+
+public class TTService implements TravelTimeService {
+  /** A String containing the earth model path for the locator, null to use default. */
+  private String modelPath = null;
+
+  /** A String containing the serialized path for the locator, null to use default. */
+  private String serializedPath = null;
+
+  /**
+   * The TTService constructor. Sets up the earth model path.
```

Comment:
TTService class should have a comment with the general class purpose or description, either here at the contructor or more ideally at the class level